### ISSUE TITLE
Difference in readings on same machine and colors with Colour Contrast Analyser and Pika

### ISSUE THREAD

**Issue by lloydi** [superhighfives/pika#136]
I was checking a recent update of the CCA tool (TPGi), and did a quick comparison. I note that despite picking the same colors (and both tools reading the same Hex value), the contrast ratio was different.

Color 1: #343a40
Color 2: #3e444a

Pika reports 1.17:1
CCA reports 1.2:1

<img width="998" alt="image" src="https://github.com/user-attachments/assets/2e42350e-bc46-4a76-9829-70daa9173cab">

Why do they differ?
More importantly, which one is correct?

For info, I have PIKA set to the sRGB color space:

<img width="310" alt="image" src="https://github.com/user-attachments/assets/64b14369-0c99-4990-9fdb-4da77677f15f">

If choose system default, the color picker hex values do not match what CCA has.

**Comment by superhighfives** [comment]
I'm not 100% sure how CCA handles working out colour contrast—you can see all the logic for how Pika compares colours here: https://github.com/superhighfives/pika/blob/main/Pika/Extensions/Cula.swift

Super happy to make changes to that logic if anything looks off, but without knowing exactly how CCA is coming to that number it's hard for me to troubleshoot.

**Comment by superhighfives** [comment]
Oh sweet, the source code is available! 👍 https://github.com/ThePacielloGroup/CCAe

Will take a look.

**Comment by lloydi** [comment]
And maybe @ferlings will know why these might differ?

**Comment by superhighfives** [comment]
Hmmm, looks like CCA doesn't handle colour spaces, so it's likely whatever that version of Electron defines as the default.

**Comment by ferllings** [comment]
No, the color space will only affect the captured color, that would have result in different hex values.
@lloydi could you check the ratio precision, on the Options panel. That might be set to 1 decimal place, and the result have been rounded.
@superhighfives btw, I like your tool, nice design
